File: vue/src/network/apply.js
import {
  POST,
  GET
} from './request.js';

export function HTTP_GETALL(data) {
  return GET({
    url: "/v1/apply/getAll",
    params: {
      page: data.page,
      limit: data.limit,
      aid: data.aid,
      state: data.state,
      name: data.name
    }
  })
}
export function HTTP_ADD(data) {
  return POST({
    url: "/v1/apply/add",
    data: {
      name: data.name,
      mold: data.mold,
      bin: data.bin,
      login: data.login,
      msg: data.msg
    }
  })
}

export function HTTP_EDIT(data) {
  return POST({
    url: "/v1/apply/edit",
    data: {
      aid: data.aid,
      name: data.name,
      mold: data.mold,
      bin: data.bin,
      login: data.login,
      msg: data.msg,
      state: data.state
    }
  })
}

File: go/app/mode/apply.go
package mode

type Apply struct {
	Aid   int    `gorm:"primary_key" json:"aid" form:"aid"`
	Name  string `gorm:"column:i_name" json:"name" form:"name"`
	Mold  int    `gorm:"column:i_mold" json:"mold" form:"mold"`
	Login int    `gorm:"column:i_login" json:"login" form:"login"`
	Bin   int    `gorm:"column:i_bin" json:"bin" form:"bin"`
	Msg   string `gorm:"column:i_msg" json:"msg" form:"msg"`
	State int    `gorm:"column:i_state" json:"state" form:"state"`
	Time  int    `gorm:"column:i_time" json:"time"`
	Page  int    `gorm:"-" form:"page" json:"-"`
	Limit int    `gorm:"-" form:"limit" json:"-"`
}
